driver.go

```diff
@@ -257,22 +258,86 @@ func findNeighbors(tx neo4j.Transaction, ids []uint64) ([]neo4jNode, []neo4jNode
 	return sources, targets, edges, nil
 }
 
-func MakeProperties(input map[string]interface{}) map[string]*ls.PropertyValue {
-	ret := make(map[string]*ls.PropertyValue)
+// for k, v := range input {
+// 	if k != cfg.Map(ls.NodeValueTerm) {
+// 		continue
+// 	}
+// vt := cfg.Map(cfg.PropertyTypes[k])
+// if vt != "" {
+// 	va := ls.GetValueAccessor(vt)
+// 	switch val := v.(type) {
+// 	case bool, neo4j.Time, neo4j.LocalTime, neo4j.LocalDateTime, neo4j.Date:
+// 		if va != nil {
+// 			form, err := va.FormatNativeValue(val, val, node)
+// 			if err != nil {
+// 				panic(fmt.Errorf("cannot format native value for %v, %w", v, err))
+// 			}
+// 			return form
+// 		} else {
+// 			return val
+// 		}
+// 	case float64:
+// 		if va != nil {
+// 			form, err := va.FormatNativeValue(val, val, node)
+// 			if err != nil {
+// 				panic(fmt.Errorf("cannot format native value for %v, %w", v, err))
+// 			}
+// 			return form
+// 		} else {
+// 			f := strconv.FormatFloat(v.(float64), 'f', -1, 64)
+// 			return f
+// 		}
+// 	default:
+// 		return v
+// 	}
+
+// Called before SetNodeValue, input contains neo4j native values
+func SetNodeValueAfterLoad(cfg Config, node graph.Node, input map[string]interface{}) interface{} {
+	// check if value type exists in config
+	v, ok := input[cfg.Map(ls.NodeValueTerm)]
+	if !ok {
+		return nil
+	}
+	return neo4jValueToNativeValue(v)
+}
+
+func BuildNodePropertiesAfterLoad(node graph.Node, input map[string]interface{}, cfg Config) {
 	for k, v := range input {
-		switch v.(type) {
-		case string:
-			ret[k] = ls.StringPropertyValue(v.(string))
-		case []interface{}:
-			isl := v.([]interface{})
-			sl := make([]string, 0, len(isl))
-			for _, val := range isl {
-				sl = append(sl, val.(string))
+		expandedKey := cfg.Expand(k)
+		node.RemoveProperty(k)
```

Comment:
If node has shortened property name here, then you are setting those after load before this. Remove